$OpenBSD: patch-test_test_api_test_completion_py,v 1.1 2019/10/14 22:31:20 kmos Exp $

the affected test tries to access:

  /path/to/src/jedi-version/../jedi

instead of

  /path/to/src/jedi-version/../jedi-version

This probably works because the default behaviour when cloning using git
creates a folder named "jedi" and not "jedi-version" as we do on build.

This is fixed upstream but still not released.

Index: test/test_api/test_completion.py
--- test/test_api/test_completion.py.orig
+++ test/test_api/test_completion.py
@@ -1,4 +1,5 @@
-from os.path import join, sep as s
+from os.path import basename, join, sep as s
+from os import getcwd
 import sys
 from textwrap import dedent
 
@@ -181,7 +182,7 @@ se = s * 2 if s == '\\' else s
         ('test%sexample.py' % se, 'r"test%scomp"' % s, 5, ['t' + s]),
         ('test%sexample.py' % se, 'r"test%scomp"' % s, 11, ['letion' + s]),
         ('test%sexample.py' % se, '"%s"' % join('test', 'completion', 'basi'), 21, ['c.py']),
-        ('example.py', 'rb"' + join('..', 'jedi', 'tes'), None, ['t' + s]),
+        ('example.py', 'rb"' + join('..', basename(getcwd()), 'tes'), None, ['t' + s]),
 
         # Absolute paths
         (None, '"' + join(root_dir, 'test', 'test_ca'), None, ['che.py"']),
